event_webhook_types.go

```diff
@@ -340,10 +339,10 @@ type IssueEvent struct {
 		TimeEstimate        int      `json:"time_estimate"`
 		TotalTimeSpent      int      `json:"total_time_spent"`
 		TimeChange          int      `json:"time_change"`
-		HumanTotalTimeSpent string   `json:"human_total_time_spent"` // TODO Check type
-		HumanTimeEstimate   string   `json:"human_time_estimate"`    // TODO Check type
-		HumanTimeChange     string   `json:"human_time_change"`      // TODO Check type
-		Weight              string   `json:"weight"`                 // TODO Check type
```

Comment:
So did you check/confirm this? Can you share your findings? Same question goes for all places where you removed similar comments.